docs/faq.md

```diff
@@ -2,41 +2,51 @@
 
 ## What is Jupytext?
 
-Jupytext is a Python package that can convert Jupyter notebooks to scripts or Markdown documents. It can also convert these text documents back to Jupyter notebooks.
+Jupytext is a Python package that provides _two-way_ conversion between Jupyter Notebooks and several other text-based formats like Markdown documents or scripts.
 
 ## Why would I want to convert my notebooks to text?
 
 The text representation have much cleaner diffs than the original notebook format. Merging multiple contributions to a notebook in any of these text formats is easier than with the JSON format. Last but not least, acting on a notebook represented as text (spell check, reformat, ...) is sometimes more comfortable than in Jupyter.
```

Comment:
One of the use cases I have for an md only version (no ipynb save) is a notebook where I am using database calls that display sensitive information as outputs, even if the db queries and code manipulations I do later on returned data are not sensitive.

By making sure I don't save the notebook, the fact that the saved md document *does not* contain code outputs (sensitive data) is a win.

Reply:
Interesting use case! We could add a mention that only the inputs are saved, that they match well what the user has effectively contributed to the notebook, and that as you say that's an effective way to drop the outputs which can be large or private.